bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,233 @@ class StylintBear:
+                        placeholder: bool=True,
+                        prefix_vars_with_dollar: bool=True,
+                        semicolons: bool=False,
+                        sort_order: str='alphabetical',
+                        stacked_properties: bool=False,
+                        check_valid_property: bool=True,
+                        zero_units: bool=False,
+                        z_index_normalize: bool=False,
+                        stylint_config: str=''):
+        """
+        :param block_keyword:
+            When ``True`` expect the ``@block`` keyword when defining block
+            variables. When ``False``, expect no ``@block`` keyword when
+            defining block variables.
+        :param brackets:
+            When ``True``, expect ``{}`` when declaring a selector. When
+            ``False``, expect no brackets when declaring a selector.
+        :param colons:
+            When ``True``, expect ``:`` when declaring a property. When
+            ``False``, expect no ``:`` when declaring a property.
+        :param color_variable_over_hex_value:
+            When ``True``, enforce variables when defining hex values.
+        :param spaces_after_commas:
+            Enforce or disallow spaces after commas.
+        :param spaces_after_comments:
+            Enforce or disallow spaces after line comments.
+        :param allow_trailing_whitespace:
+            If ``True``, ignores trailing white space. If ``False``, trailing
+            white space will throw a warning.
+        :param css_literal:
+            By default Stylint ignores ``@css`` blocks. If set to ``True``
+            however, it will throw a warning if ``@css`` is used.
+        :param max_selector_depth:
+            Set the max selector depth. If set to 4, max selector depth will
+            be 4 indents. Pseudo selectors like ``&:first-child`` or
+            ``&:hover`` won't count towards the limit.
+        :param check_duplicates:
+            Checks if selector or property duplicated unnecessarily.
+        :param efficient_properties:
+            Check for places where properties can be written more efficiently.
+        :param extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+            For example: If set to ``@extends``, prefer
+            ``@extends $some-var`` instead of ``@extend $some-var``
+            or if set to ``@extend``, prefer
+            ``@extend $some-var`` over ``@extends $some-var``.
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``0``.
+            For example: If set to ``2``, prefer
+            ``/s/smargin: 0`` over ``/s/s/smargin: 0``.
```

Comment:
Nope, this is just so the user can differenciate between tabs and spaces and see how the setting works.